spec/haskell/src/SEL4/API/Invocation/AARCH64.hs

```diff
@@ -114,6 +123,11 @@ data IRQControlInvocation
         issueHandlerSlot,
         issueHandlerControllerSlot :: PPtr CTE,
         issueHandlerTrigger :: Bool }
+    | IssueSGISignal {
+        issueSGIIRQ :: Word,
+        issueSGITargets :: Word,
+        issueSGIControlSlot,
+        issueSGISlot :: PPtr CTE }
```

Comment:
Do we want any more documentation here? I can guess at IRQ and Targets, but ControlSlot and Slot are completely baffling to me.